src/test/scala/org/graphframes/PregelSuite.scala

```diff
@@ -58,14 +58,14 @@ class PregelSuite extends SparkFunSuite with GraphFrameTestSparkContext {
     val pageRankResultDF = new Pregel(graph1)
       .setMaxIter(15)
       .withVertexColumn("rank", lit(1.0 / N),
-        when(Pregel.msg.isNotNull, Pregel.msg).otherwise(col("rank")))
+        when(Pregel.msg.isNotNull, Pregel.msg).otherwise(lit(0.0)))
```

Comment:
`coalesce(Pregel.msg, lit(0.0))`

Reply:
I use `(1.0 - alpha) / N` instead of zero when recv no msgs.
because the page rank update formula is `new_pr = sum(msgs) * alpha + (1 - alpha) / N`